src/components/NestedEditComponent.tsx

```diff
@@ -302,17 +302,25 @@ export class NestedEditComponent extends React.Component<InlineComponentProps, a
     }
 
     updateDefaultValue = (props: any) => {
-        const defaultValue: any = {}
+        const newValue: any = {}
         map(props.field.fields, field => {
             if (field.hasOwnProperty("defaultValueFn") && (!props.currentModel || !props.currentModel.hasOwnProperty(field.field))) {
-                defaultValue[field.field] = field.defaultValueFn()
+                newValue[field.field] = field.defaultValueFn(props)
+            }
+            
+            if (field.hasOwnProperty("valueFn")) {
+                let resolvedValue = field.valueFn({ field, model: props.currentModel })
```

Comment:
can we just pass `props` here? 
`field` will already be available in the scope where you are defining valueFn I guess.